src/components/app/app.js

```diff
@@ -99,6 +99,8 @@ App.propTypes = {
     exportBtn: PropTypes.bool,
     pipelineBtn: PropTypes.bool,
     sidebar: PropTypes.bool,
+    zoomToolBar: PropTypes.bool,
+    miniMapBtn: PropTypes.bool,
```

Comment:
@jitu5 I remember we thought of having miniMap only in display which will control both the button and the actual rectangular map area. Was there any change later ?